Makefile

```diff
@@ -100,12 +100,16 @@ kind-cluster-cleanup: kind ## Delete the kind cluster
 ###################
 # Install and Run #
 ###################
-.PHONY: install run cert-mgr
+.PHONY: install install-manifests wait run cert-mgr uninstall
 
 ##@ install/run:
 
-install: generate cert-mgr ## Install rukpak
+install: generate cert-mgr install-manifests wait ## Install rukpak
+
+install-manifests:
```

Comment:
Note: I didn't add a comment for this (or the wait) target as they seem like "internal" commands from the user's perspective.

Reply:
I agree with that.